### PR TITLE
Adding new program

### PR DESCRIPTION
**Pull request by sarthakbal0987**
I  added a java program to check whether the  given number entered by the user is an armstrong number or not

### CODE REVIEW

**Comment by Dev-Soumyaranjan** [comment]
@all-contributors please add @sarthakbal0987  for code

**Comment by Dev-Soumyaranjan** [comment]
@all-contributors please add @sarthakbal0987   for code

File: PrepInsta Top 100 Codes/Armstrong_Number.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
```
Comment (Dev-Soumyaranjan) [inline PrepInsta Top 100 Codes/Armstrong_Number.java]: Add program description here.

File: PrepInsta Top 100 Codes/Armstrong_Number.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+public class Armstrong_Number {
+     public static void main(String[] args) {
+        // first we have to give a input number using the Scanner class also we have to import the package of Scanner class
```
Comment (Dev-Soumyaranjan) [inline PrepInsta Top 100 Codes/Armstrong_Number.java]: Try to make your comment short and efficient

File: PrepInsta Top 100 Codes/Armstrong_Number.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        int sum=0;
+        // we have to declare the variable 'rem' in which we will store the remainder of the division of 'n' by 10
+        int rem;
+        // now we will start our while loop to get the sum of the cube of each digit of 'n'
+        while(n>0)
+        {
+            // we have to store the remainder of the division of 'n' by 10 in the variable 'rem'
+            rem=n%10;
+            // we have to store the cube of 'rem' in the variable 'sum'
+            sum=sum+(rem*rem*rem);
```
Comment (Dev-Soumyaranjan) [inline PrepInsta Top 100 Codes/Armstrong_Number.java]: According to wikipedia Armstrong number is a number that is the sum of its own digits each raised to the power of the number of digits.
But you have applied different logic here.
Yeah it's true that in some website armstrong number logic is exactly same as yours but i prefer to follow the wikipedia so i request you to change this logic.

**Review by Dev-Soumyaranjan — CHANGES_REQUESTED** [review]
I have added some points, please address those points before this request gets merged

File: PrepInsta Top 100 Codes/Armstrong_Number.java (hunk 2 above)
Comment (sarthakbal0987) [reply PrepInsta Top 100 Codes/Armstrong_Number.java]: ok sure

File: PrepInsta Top 100 Codes/Armstrong_Number.java (hunk 3 above)
Comment (sarthakbal0987) [reply PrepInsta Top 100 Codes/Armstrong_Number.java]: Yeah , now i got your point .

**Pull request closed without merging**